src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -449,9 +450,7 @@ private function autoPrependText(): void
 
     private function writeBuffer(string $message, bool $newLine, int $type): void
     {
-        // We need to know if the two last chars are PHP_EOL
-        // Preserve the last 4 chars inserted (PHP_EOL on windows is two chars) in the history buffer
```

Comment:
maybe this comment needs to be preserved somewhere, to explain why we have this buffer